website/docs/filter-group/add-filter/examples/add-filter.tsx

```diff
@@ -0,0 +1,223 @@
+import React from 'react';
+import DropdownMenu from '@semcore/dropdown-menu';
+import Select from '@semcore/select';
+import Input from '@semcore/input';
+import MathPlusM from '@semcore/icon/MathPlus/m';
+import { Hint } from '@semcore/ui/tooltip';
+import { Flex } from '@semcore/flex-box';
+import Button, { ButtonLink } from '@semcore/button';
+import { FilterTrigger } from '@semcore/base-trigger';
+import CloseM from '@semcore/icon/Close/m';
+import SearchM from '@semcore/icon/Search/m';
+
+type FilterSelectProps = {
+  value: string;
+  options: string[];
+  onClear: () => void;
+  onChange: (value: string) => void;
+  name: string;
+};
+
+const FilterSelect = ({ value, onClear, onChange, options, name }: FilterSelectProps) => {
+  const selectPopperRef = React.useRef<HTMLDivElement>();
+
+  const isEscapeKeyDown = (e) => {
+    return e.key === 'Escape';
+  };
+
+  return (
+    <Select interaction={'focus'} value={value} onChange={onChange} placeholder={name}>
```

Comment:
```suggestion
    <Select value={value} onChange={onChange} placeholder={name}>
```
interaction should be default. If you added interaction=focus to open the filter after adding, I think we have to do it some other way

Reply:
Implemented another approach to focus on element, when select is opened.